astropy/tests/helper.py

```diff
@@ -460,53 +459,10 @@ def assert_quantity_allclose(actual, desired, rtol=1.e-7, atol=None,
     This is a :class:`~astropy.units.Quantity`-aware version of
     :func:`numpy.testing.assert_allclose`.
     """
-    import numpy as np
-    np.testing.assert_allclose(*_unquantify_allclose_arguments(actual, desired,
```

Comment:
It seems this line, which was reinstated because of a comment by @mhvk got deleted again in the sphinx fixes. @pllim can we put this back in your second PR, to fix @cdeil's problem?

Reply:
Ops. I only saw Tom's comment. Sorry. I'll fix this shortly.